src/js/tracks/text-track-settings.js

```diff
@@ -321,17 +323,12 @@ class TextTrackSettings extends Component {
    * @method restoreSettings
    */
   restoreSettings() {
-    let err;
     let values;
 
     try {
-      [err, values] = safeParseTuple(window.localStorage.getItem('vjs-text-track-settings'));
```

Comment:
I don't know that it does allow us to easily differentiate. Either way, you have an `Error` object that gets logged to the console. I suppose before it was sometimes logged as an error and sometimes as a warning, but... that doesn't seem like a significant benefit.

It's used in the `Player`. I dunno if it's worth including it at all since it's basically just a small wrapper around `try`/`catch`... but that's for another discussion/PR.